test/specs/runApexTests.e2e.ts

```diff
@@ -255,7 +260,26 @@ describe('Run Apex Tests', async () => {
     const apexTestsSection = await sidebarView.getSection('APEX TESTS');
     expect(apexTestsSection.elem).toBePresent();
 
-    const apexTestsItems = (await apexTestsSection.getVisibleItems()) as TreeItem[];
+    let apexTestsItems = (await apexTestsSection.getVisibleItems()) as TreeItem[];
+
+    // If the Apex tests did not show up, click the refresh button on the top right corner of the Test sidebar
+    for (let x = 0; x < 3; x++) {
```

Comment:
I think we should just refresh like just in case

Reply:
At first, I tried to do the refresh just once, but it happened to me a few times that doing the refresh once still kept the value of apexTestsItems at 1 instead of changing it to 6.  Therefore, I decided to add a for loop to refresh up to 3 times.  (The check in the else if block is there so that it doesn't always refresh 3 times if that's not needed.)  I found that after I added the for loop, my runs on remote stopped failing at this place.